skyportal/models/observation.py

```diff
@@ -0,0 +1,45 @@
+__all__ = ['ExecutedObservation']
+
+import sqlalchemy as sa
+from sqlalchemy.orm import relationship
+
+from baselayer.app.models import Base
+
+
+class ExecutedObservation(Base):
+    """Observation information, including the field ID, exposure time, and
+    filter."""
+
+    instrument_id = sa.Column(
+        sa.ForeignKey('instruments.id', ondelete="CASCADE"),
+        nullable=False,
+        doc='Instrument ID',
+    )
+
+    instrument_field_id = sa.Column(
+        sa.ForeignKey("instrumentfields.id", ondelete="CASCADE"),
+        primary_key=True,
+        doc='Field ID',
+    )
+
+    observation_id = sa.Column(sa.Integer, primary_key=True, doc='Observation ID')
```

Comment:
Isn't there already an `id` field inherited from `Base` in baselayer?

Reply:
OK I understand this is an external ID from the observer side. Maybe make that clearer that this is a user supplied ID, different than the internal database ID.

Reply:
extending the doc here a bit.